docs/UserGuide.md

```diff
@@ -32,80 +32,173 @@
 > **Command Format**
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
-> * The order of parameters is fixed.
+> * The order of parameters is fixed, with optional parameters being the exception.
+> * Parameters `START_DATE` and `END_DATE` has to be in dd-mm-yyyy format (e.g. 23-10-2016).
+> * Parameters `START_TIME` and `END_TIME` has to be in 24 hour format (e.g. 1215).
+> * Parameter `TIMELEFT` **must be a positive integer**, indicating number of hours left.
 
 #### Viewing help : `help`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a task : `add`
+Adds a task to the task manager.<br>
+Format: `add TASK_NAME [sd/START_DATE] [st/START_TIME] [ed/END_DATE] [et/END_TIME] [tl/TIMELEFT]` 
 
-> Persons can have any number of tags (including 0)
+> `TASK_NAME` is required, but the rest of the parameters are optional.<br>
+  User can specify `END_DATE` or `TIME_LEFT` to set a deadline to a task.<br>
+  User can specify `START_DATE`, `START_TIME`, `END_DATE` and `END_TIME` to set an event to the task.
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `add follow up with Jack on sales report sd/6-10-2016`
+* `add project Highlight sd/1-10-2016 ed/14-11-2016`
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+#### Set a deadline to a task : `setdeadline`
+Sets the amount of time left or an end date to the specified task from the task manager.<br>
+Format: `setdeadline TASK_INDEX tl/TIMELEFT` or `setdeadline TASK_INDEX ed/END_DATE`
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+> Sets a deadline to the task at the specified `TASK_INDEX`.<br>
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
+  
+Examples: 
+* `list`<br>
+  `setdeadline 2 tl/24`<br>
+  Set a deadline with 24 hours left to the 2nd task in the task manager.
+* `find report`<br> 
+  `setdeadline 1 ed/23-10-2016`<br>
+  Set a deadline on 23rd October 2016 to the 1st task in the results of the `find` command.
+
+#### Set an event to a task : `setevent`
+Sets an event to the specified task from the task manager.<br>
```

Comment:
I remember the discussion during the tutorial on setting "event to" and "deadline to" the task.
Since it is not the common way of using task manager, it would help to explain a bit about the task structure in the beginning.